src/test/java/handlers/CameraHandlerTest.java

```diff
@@ -60,8 +59,7 @@ void addCameraTest() {
     @Test
     void getNewFrameTest() {
```

Comment:
de methode getnewframe is weggehaald toch, misschien moeten deze methode naam + javadoc dan aangepast worden naar wat het daadwerkelijk test.

Reply:
Done